lib/mix/test/mix/tasks/compile_test.exs

```diff
@@ -9,6 +9,12 @@ defmodule Mix.Tasks.CompileTest do
     end
   end
 
+  defmodule WrongPath do
+    def project do
+      [app: :apps_path_bug, apps_path: "this_path_does_not_exist_or_is_empty"]
```

Comment:
It's wrong the part of the atom that says its_empty. Maybe another module with empty path can be created and tested

Reply:
Yes i'll create